podman-image/Containerfile

```diff
@@ -51,18 +51,24 @@ RUN if [[ ${PODMAN_PR_NUM} == "" ]]; then \
         rpm-ostree override replace --experimental --freeze \
         --from repo="copr:copr.fedorainfracloud.org:packit:containers-podman-${PODMAN_PR_NUM}" \
         podman; \
-    fi && \
-    rpm-ostree override remove moby-engine containerd runc && \
-    rm -fr /var/cache && \
-    ostree container commit
+    fi
 
 # Install subscription-manager and enable service to refresh certificates
 # Install qemu-user-static for bootc
 # Install gvisor-tap-vsock-gvforwarder for hyperv
+# Install device-mapper (this satisfies the deps for qemu-user-static
 # Install ansible for post-install configuration
-RUN rpm-ostree install subscription-manager gvisor-tap-vsock-gvforwarder qemu-user-static ansible-core && \
+# Remove unwanted packages
+# Remove man pages (man binary is not present)
+RUN dnf install -y --setopt=install_weak_deps=false \
+    subscription-manager device-mapper qemu-user-static-aarch64 qemu-user-static-x86 && \
+    dnf install -y gvisor-tap-vsock-gvforwarder ansible-core && \
+    dnf remove -y moby-engine containerd runc toolbox qed-firmware docker-cli && \
     rm -fr /var/cache && \
-    ostree container commit
+    dnf -y clean all && \
+    ostree container commit && \
+    rm -fr /usr/share/man
```

Comment:
Is there a reason to do the man page removal after the `ostree container commit` ? maybe it should be just another argument to the earlier `rm -rf`